### PR TITLE
Add supply chain security (s3c) working group to list 👷‍♀️

### PR DESCRIPTION
**Pull request by bobcatfish**
Last week we had the first meeting of a working group focused on cross
project supply chain security interests, this commit adds the details
about this meeting going forward to the list of working groups.

We might want to replace the agenda doc and slack links with something
else if we're not happy with those but this at least gets something in
place :D

### CODE REVIEW

**Comment by bobcatfish** [comment]
@pritidesai @imjasonh @afrittoli @xchapter7x based on our discussions last week I've put you all (and myself) into the facilitator rotation, just say the word if you'd rather not be

File: working-groups.md [hunk 1]
```diff
@@ -385,3 +386,25 @@ and a place to discuss community operations and process.
...
+
+The goal of this working group is to discuss supply chain security initiatives across Tekton (exact scope 
+TBD [community#629](https://github.com/tektoncd/community/issues/629)).
+
+| Artifact                   | Link                       |
+| -------------------------- | -------------------------- |
+| Forum                      | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) |
+| Meeting Notes              | [Discussion board](https://github.com/orgs/tektoncd/projects/14/views/1), [HackMD Notes](https://hackmd.io/gFcAZMFMRwuTaZ1i7Y3fSg)
+| Slack Channel              | [#security](https://tektoncd.slack.com/messages/security) |
+| Community Meeting VC       | [meet.google.com/utk-kdff-hci](https://meet.google.com/utk-kdff-hci) |
```
Comment (afrittoli) [inline working-groups.md:400]: I setup a zoom meeting as discussed https://zoom.us/j/96593435267?pwd=TTNVYUJEQlNzMXlKYjFXcUwzOUZEdz09

Comment (afrittoli) [reply working-groups.md:400]: See https://github.com/tektoncd/community/issues/630

Comment (bobcatfish) [reply working-groups.md]: Thanks @afrittoli ! I've updated the link here

**Review by dibyom — APPROVED** [review]

**Review by jerop — APPROVED** [review]
/lgtm

**Pull request merged**